DigitalOffice/Services/UserService/Validators/EditUserRequestValidator.cs

```diff
@@ -0,0 +1,57 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.UserService.RestRequests;
+
+namespace LT.DigitalOffice.UserService.Validators
+{
+    public class EditUserRequestValidator : AbstractValidator<EditUserRequest>
+    {
+        public EditUserRequestValidator()
```

Comment:
Hmm, we have the same checks when creating a user. Not good (DRY)

Reply:
Agree. But is it good to use UserCreateRequestValidator in edit user?

Reply:
It is not necessary to use it, you can separate

Reply:
(maybe there is a solution here) link